### ISSUE TITLE
Paper 6.12 Error with Hugo v0.112.3

### ISSUE THREAD

**Issue by WillGreen** [nanxiaobei/hugo-paper#166]
Hello,

I recently updated Hugo from 0.111.3 -> 0.112.3 on macOS (darwin/arm64). 

With Paper 6.12, I'm now seeing the following error `Get is not a method but has arguments`:

```shell
$ hugo       
Start building sites … 
hugo v0.112.3+extended darwin/arm64 BuildDate=unknown
ERROR 2023/05/26 16:05:55 render of "page" failed: "***/layouts/_default/baseof.html:41:7": execute of template failed: template: _default/single.html:41:7: executing "_default/single.html" at <partial "header.html" .>: error calling partial: "***/themes/paper/layouts/partials/header.html:89:144": execute of template failed: template: partials/header.html:89:144: executing "partials/header.html" at <site>: Get is not a method but has arguments
...
```

Line 89 of `layouts/partials/header.html` is:

```plaintext
href="{{ if eq . `rss` }}{{ `index.xml` | absURL }}{{ else if eq . `mastodon` }}{{ site.Params.Get . }}{{ else }}https://{{ . }}.com/{{ site.Params.Get . }}{{ end }}"
```

NB. `***` in the error output is me trimming the long path.

**Comment by nanxiaobei** [comment]
Fixed in `6.13.0`.

https://github.com/gohugoio/hugo/issues/10862

**Comment by WillGreen** [comment]
Thank you for the fast fix and all your hard work on Paper.
